### ISSUE TITLE
Docs: docker-machine upgrade DockerHostName

### ISSUE THREAD

**Issue by superherointj** [docker/docker.github.io#3381]
File: [machine/reference/upgrade.md](https://docs.docker.com/machine/reference/upgrade/), CC @londoncalling 

Hi. 
How do I upgrade a DockerHost when it fails on ¨docker-machine upgrade xxx¨ with error: 
¨Boot2Docker URL was explicitly set to ¨XXXXXXX¨ at create time, so Docker Machine cannot upgrade this machine to the latest version.¨
Docs doesn't cover anything related to it.

**Comment by mstanleyjones** [comment]
Could this relate to https://github.com/docker/machine/issues/491

**Comment by johndmulhausen** [comment]
@superherointj We're unclear on whether you're trying to upgrade to the latest boot2docker or specifying your machine name in the 'xxx' portion of your example command. If you specify a machine name it will upgrade to the latest boot2docker; there isn't support for upgrading to specific versions.

**Comment by superherointj** [comment]
I did gave up on solving this. The answer took too long. But I did figure out why things didn't work out right.
The issue was the dockerhost being created by docker-machine using vmware driver WITH a specific ISO specified. This made the upgrade fail. So, it was not possible to upgrade a dockerhost that had been created using a specific ISO.
At the end, I just gave up upgrading the dockerhost and had to recreate everything from scratch.
